test/testpaint/Compat.cpp

```diff
@@ -38,15 +38,15 @@ uint32_t gScenarioTicks;
 uint8_t gCurrentRotation;
 
 // clang-format off
-const CoordsXY CoordsDirectionDelta[] = {
-    { -32, 0 },
-    { 0, +32 },
-    { +32, 0 },
-    { 0, -32 },
-    { -32, +32 },
-    { +32, +32 },
-    { +32, -32 },
-    { -32, -32 },
+static constexpr const std::array<CoordsXY, 8> CoordsDirectionDelta = {
```

Comment:
```suggestion
constexpr const std::array<CoordsXY, 8> CoordsDirectionDelta = {
```

It can't be static, as that violates the declaration in `TileElement.h`.

Reply:
Correct, changed it,